### PR TITLE
Fix: issue #98 restyle close button + close animation

### PR DESCRIPTION
**Pull request by lexich**
#98

### CODE REVIEW

**Comment by Guria** [comment]
Close button isn't accessible from a keyboard. Please make it focusable and focus it when panel is opened

File: web/app/components/user-info/user-info.jsx [hunk 1]
```diff
@@ -41,10 +41,11 @@ class UserInfo extends Component {
         <p className="user-info__id">{id}</p>
 
         <LastCommentsList isLoading={isLoading} comments={comments} />
-
-        <span {...getHandleClickProps(onClose)} className="user-info__close">
-          Close &#10006;
-        </span>
+        {!onClose ? null : (
```
Comment (Guria) [inline web/app/components/user-info/user-info.jsx]: It seems it is unused now.

**Comment by lexich** [comment]
@Guria Pls check PR.

**Comment by lexich** [comment]
Yes, my fault. Fixed.

**Pull request merged**